### ISSUE TITLE
First player always red and clocks

### ISSUE THREAD

**Issue by dotdotdot-tts** [rogeriocamorim/duneBugSuggestionTracker#27]
low priority features - is it on the road map to bringing back:
- "first player always red"
- clocks

these were popular features added based on community feedback in the past.

**Comment by aaittouares** [comment]
"first player always red" 
won't happen: it was for purposes that are not needed anymore ( stats were based on first player, and there was no way to identify 1st player in the data uploaded to server). When data will happen we will export the data for who is first player.

Clocks are very low priority for us, a lot of other stuff need to be done before and after that we need to think about what space is available for putting some clocks.
Even in Dune + ix + immortality ranked and tournaments clocks are not enforced.
It's a "nice to have" thing but also split community between "core" and new players. 
Since Uprising is still new, i prefer not to implement that yet and dedicate time to more urgent matters.

**Comment by dotdotdot-tts** [comment]
ah yeah that is fair on both points. 
- it may be a minor thing but i will say that mental processing-wise, it was nice to know that if you're red, you go first. also knowing blue always picking leader first as well. i believe, it is a bit more functional than just the stat gathering it was originally designed for. 
- i agree clocks can get divisive, especially if players get abusive over it. it still is a nice feature to help set an expectation of game length so it doesn't get too long. some tournaments have enforced clock, salty throne and the muad'dib melees.

**Comment by elessar-gh** [comment]
Agree with Dot that Red Goes First is about more than just stats. It was a nice tradition that makes it easier to track what round it is, both as a spectator and as a player. FWIW I'm a big supporter of bringing it back :)

**Comment by harry71019** [comment]
Yeah I agree it was overall a good way to know which round it was. Not just from a player point of view, but also for people watching content whether live or recording on youtube. You could directly tell which round it was and the state without having to look at the combat deck (which you can't do when watching a video).
Maybe something that could indicate which round it is would be a nice update?

**Comment by scmccarthy** [comment]
I still miss red going first every game I play. (Well, I guess only about 3/4 of the games I play.)

**Comment by dotdotdot-tts** [comment]
@aaittouares please reconsider the "red goes first" feature. it is really missed by the community. it is much more functional for both playing and spectating as players have expressed on here and on discord.

**Comment by sseny1** [comment]
Red going first wasn't for stat tracking. That was just a secondary benefit. It was originally a function to be able to sit in specific seating positions for the Water of Life tournament. We ended up realizing that it was easier to visually track information when you're playing or watching a game, though. This is why we want it back. It's actually a quality-of-life improvement.

**Comment by aaittouares** [comment]
" It was originally a function to be able to sit in specific seating positions for the Water of Life tournament."
Totally true

Also the benefit was that apparently you could deduce what round you were playing.

So the need apparently is NOT "i want to have red go first" (cause you won't because it would impact how the objectives cards are handled and that part is fine right now) but "i want to be able to see what round is currently going".

If that is the need then we can try to put some clear info about that.

**Comment by scmccarthy** [comment]
You can *deduce* what round you're playing simply by looking at how many conflict cards are left. The benefit is being able to *see* it, all the time, effortlessly. And to also see who has which start position without having to rotate things in your mind.

For example, you commonly want to know who will go first in round 7. (Answer: Yellow.)

Really, I promise, red going first is the feature being requested!

**Comment by aaittouares** [comment]
@scmccarthy  then it won't happen if it's "red going first"

You can ask all you want, that doesn't mean i will say yes ;)

I told you : your need is one thing and the technical solution is another

"Red going first" is a technical solution that i see as a problem for the project.
Your need is : "i want to be able to see what round i'm playing at a glance" and also "i want to be able to see starting positions at a glance"

it is not "i want to have red going first" ( and what does red go first alone means by the way in that context ??) 

If you accept that we (the team) design a solution (wich will not be "red go first") to answer those 2 needs i described above, then maybe we can proceed.
Else, i will consider closing the ticket cause this won't advance.

**Comment by scmccarthy** [comment]
You can obviously do what you want - I'm only clarifying what the request is.

**Comment by scmccarthy** [comment]
If it helps, the request is really that the start player token go to a fixed position, such as top left.

**Comment by aaittouares** [comment]
@scmccarthy 
So to be clear: is the request what i described ??

1 - "i want to be able to see what round i'm playing at a glance" 
2-  "i want to be able to see starting positions of a round at a glance"

**Comment by aaittouares** [comment]
> If it helps, the request is really that the start player token go to a fixed position, such as top left.

that doesn't help at all.
Your solution to your problem create other problems (randomization of objectives changing and impacting stats of objectives)

**Comment by scmccarthy** [comment]
Interesting, can we talk about that? what would be the issue with always dealing the start player objective to the same seat, and randomizing the rest?

**Comment by aaittouares** [comment]
> Interesting, can we talk about that? what would be the issue with always dealing the start player objective to the same seat, and randomizing the rest?

no we can't.
please answer if the request is what i described and stop trying to enforce a technical solution you think will fit.
I already told "red go first" won't happen.
I have other solutions that are fine too and won't impact other stuff in the project.

I have limited time actually and would prefer to focus on other stuff.

**Comment by scmccarthy** [comment]
I think I already told you my opinion. (I don't think it should matter that much what my opinion is, anyway, so I feel doubly silly answering again.)

Your description of it sounds to me to be aiming in the right direction but I am skeptical that another solution can succeed as well.

**Comment by dotdotdot-tts** [comment]
> So to be clear: is the request what i described ??
> 
> 1 - "i want to be able to see what round i'm playing at a glance" 2- "i want to be able to see starting positions of a round at a glance"

i would say that it is more than those 2 points. 

the consistency of the colors is the main point. it allows for the players and spectators to understand the game states much faster. this allows games to operate more smoothly and a better experience for spectators and commentators as well.

**Comment by dotdotdot-tts** [comment]
i would go as far to say, even if you gave us an alternative way to see:
- what the current round is at a glance
- the starting positions of a round at a glance

it won't make the game more clear than the consistency of red going first.

**Comment by rogeriocamorim** [comment]
Add on today's release.
![image](https://github.com/user-attachments/assets/f1b4cd89-134e-424e-bdfa-d15419426549)

Clocks... Not a big fan.....

**Comment by dotdotdot-tts** [comment]
YASSSSSSS red goes first! @Chatanga @aaittouares @rogeriocamorim

amazing work on this update, it's so good and so smooth!

THANK YOU! <3
